package/rtcm3_in_protocol/rtcm3_protocol.mk

```diff
@@ -0,0 +1,31 @@
+################################################################################
```

Comment:
can we rename this to `rtcm3_in_protocol.mk` for consistency?

Reply:
👍